package.json

```diff
@@ -70,6 +78,17 @@
                 {
                     "command": "local-history.viewHistory",
                     "when": "editorTextFocus"
+                },
+                {
+                    "command": "local-history.clearHistory",
+                    "when": "editorTextFocus"
+                }
+            ],
+            "explorer/context": [
+                {
+                    "command": "local-history.clearHistory",
+                    "when": "!explorerResourceIsFolder",
+                    "group": "CustomGroup@1"
```

Comment:
[minor] for the group, should we have a local-history group instead to keep all commands together?
In the future, we can have a command which executes the 'view history' for the selected explorer node.

Reply:
@vince-fugnitto Yes, I think we should have a local-history group in the explorer's context menu.

> In the future, we can have a command which executes the 'view history' for the selected explorer node.
https://github.com/vince-fugnitto/local-history-ext/issues/15